backend/migrations/sqls/20210928133243-testdaten-down.sql

```diff
@@ -0,0 +1,9 @@
+DELETE FROM abiturpuefungen WHERE id IN (1,2);
+DELETE FROM ag2Nutzer WHERE id IN (1,2);
+DELETE FROM ags WHERE id IN (1,2);
+DELETE FROM bezugsfach WHERE id IN (1,2);
+DELETE FROM facharbeiten WHERE id IN (1,2);
+DELETE FROM komponenten WHERE id IN (1,2);
+DELETE FROM nutzer WHERE id IN (1,2);
+DELETE FROM wahlpflichtfaecher WHERE id IN (1,2);
+DELETE FROM wahlpflichtfaecher2nutzer WHERE id IN (1,2);
```

Comment:
Warum nur die mit ID 1 und 2 löschen und nicht alle? 
Zudem fängst du hier mit id 1 an und in testdaten-up mit id 0.

Reply:
ID 1 und 2 werden gelöscht, da ich zu jeder Tabelle zwei Testdatensätze erstellt habe. So werden, wenn manuell weitere Datensätze hinzugefügt wurden diese nicht gelöscht, wenn man die Testdaten wieder entfernen möchte.

Ist das sinnvoll oder übersehe ich was?

Und richtig ich muss das in Testdaten up noch anpassen ... sqlite fängt bei 1 an ids zu vergeben

Reply:
Ist so i.O. ich habs nur mit einer "Clear DB" Funktion verwechselt, aber es geht ja wirklich nur darum, die Testdaten zu entfernen.